src/pal/src/exception/seh-unwind.cpp

```diff
@@ -203,10 +203,12 @@ static DWORD64 GetPc(CONTEXT *context)
 {
 #if defined(_AMD64_)
     return context->Rip;
-#elif defined(_ARM64_)
+#elif defined(_ARM64_) || defined(_ARM_)
     return context->Pc;
 #else
-#error don't know how to get the program counter for this architecture
+    unreached();
```

Comment:
It's not really valid code on any architecture that has the libunwind semantics of apple/arm64 unwinder.  I mean, arm/linux was never going to call this function.